software/wireshark/en.md

```diff
@@ -0,0 +1,27 @@
++++
+title = "Wireshark"
+lastmod = "2018-08-10T19:35:05+02:00"
++++
+# Wireshark
+[Wireshark](https://www.wireshark.org/) is a widely-used network protocol analyzer that lets you see what’s happening on a network at a microscopic level.
+
+## Installation
+
+Wireshark can be installed either from the Software Center or via terminal:
+
+``` bash
+sudo eopkg it wireshark
+```
+
+## Use wireshark as a normal user
```

Comment:
Still didn't address my issue with it being "normal user" instead of "non-root user"